vald/evm/evm.go

```diff
@@ -253,6 +274,44 @@ func (mgr Mgr) ProcessTransferKeyConfirmation(event *types.ConfirmKeyTransferSta
 	return err
 }
 
+func (mgr Mgr) isStuckFileCoinKeyRotation(event *types.ConfirmKeyTransferStarted) bool {
+	if mgr.chainID != "axelar-testnet-lisbon-3" {
+		return false
+	}
+	if event.PollID.String() != "1342934" {
+		return false
+	}
+
+	if event.Chain != "filecoin-2" {
+		return false
+	}
+
+	if !strings.EqualFold(event.TxID.Hex(), "0xd9cc8d5f0238dfe360776ed9029835d8a470d4d30e46a43deb76f316c0bd5740") {
+		return false
+	}
+
+	if !strings.EqualFold(event.GatewayAddress.Hex(), "0x999117D44220F33e0441fbAb2A5aDB8FF485c54D") {
+		return false
+	}
+
+	if event.ConfirmationHeight != 10 {
```

Comment:
why check this? It's not relevant for recovering the stuck tx. Also, we actually just changed this param on testnet.

Reply:
I didn't realize we couldn't retry the exact call from before. So yes, this is unnecessary